### PR TITLE
Toaster: Add test for deleting an updateable toast

### PR DESCRIPTION
**Pull request by ymouzakis**
**This change:** (check at least one)

- [ ] Adds a new feature
- [ ] Fixes a bug
- [x] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [ ] Changeset added?
- [ ] Component status page up to date?

**What does this change address?**
This PR adds a unit test that verifies an updatable toast can be deleted and removed from the dom.

**How does this change work?**
Create a toast, update it, try to delete it and verify to toaster has no toast.

**Additional context**
Add any other context here.

### CODE REVIEW

File: .changeset/young-steaks-explode.md
Comment (daneah) [inline .changeset/young-steaks-explode.md:1]: Let's delete this changeset file, as the change has no effect on the consumer experience for Pharos (i.e. there's nothing to "release" here)

Comment (michael-iden) [reply .changeset/young-steaks-explode.md:1]: It looks like maybe the wrong changeset file was removed?

Comment (daneah) [reply .changeset/young-steaks-explode.md:1]: Yes, good catch

Comment (ymouzakis) [reply .changeset/young-steaks-explode.md:1]: Woopsie!

**Review by michael-iden — APPROVED** [review]

**Review by daneah — APPROVED** [review]

**Pull request merged**